armada-ths/ais
# External signup

(This works on ais2)
Test: 
1. Make sure you are logged out.
2. Go to fairs/2017/banquet/signup. You should get redirected to register/external/signup
3. Try registering using an email that is already in use. Should get a form error.
4.  Try registering a new account. You should get redirected back to fairs/2017/banquet/signup.
5. Signup for the banquet!
6. Log out again and go back to fairs/2017/banquet/signup. You will once again be redirected to register/external/signup.
7. Click on one of the login options and login. For both you should be redirected to fairs/2017/banquet/signup when you've logged in. IMPORTANT: you can not use the kth login locally!
